driver.go

```diff
@@ -85,9 +86,19 @@ func setupLinodeAPI(token string) *linodego.Client {
 }
 
 func (driver *linodeVolumeDriver) determineLinodeID() error {
+	if err := driver.determineLinodeIDFromMetadata(); err == nil {
```

Comment:
Related to my other comment, we may want to filter errors here so we can avoid accidentally suppressing valid errors :+1: